src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -473,6 +473,33 @@ Value emitScalarOpFor<ONNXCastOp>(ConversionPatternRewriter &rewriter,
   return create.math.cast(elementType, scalarOperands[0]);
 }
 
+//===----------------------------------------------------------------------===//
+// Scalar binary ops for lowering ONNXCastLikeOp
+//===----------------------------------------------------------------------===//
+template <>
+struct ScalarOp<ONNXCastLikeOp> {
+  using FOp = CustomScalarOp;
+  using IOp = CustomScalarOp;
+};
+
+template <>
+Value emitScalarOpFor<ONNXCastLikeOp>(ConversionPatternRewriter &rewriter,
+    Location loc, Operation *op, Type elementType,
+    ArrayRef<Value> scalarOperands) {
+
+  Value input = scalarOperands[0];
+  Value target = scalarOperands[1];
+
+  Type targetType = target.getType();
+
+  CheckIfCustomScalarOpIsSupported<ONNXCastOp>(targetType);
```

Comment:
ONNXCastLike?

Reply:
Oops...my fault. That was from copying and pasting haha

Reply:
Fixed!